packages/frontend-app-api/report.api.md

```diff
@@ -29,10 +32,19 @@ export function createSpecializedApp(options?: {
   features?: FrontendFeature[];
   config?: ConfigApi;
   bindRoutes?(context: { bind: CreateAppRouteBinder }): void;
+  apis?: ApiHolder;
+  extensionFactoryMiddleware?: ExtensionFactoryMiddleware;
 }): {
+  apis: ApiHolder;
   createRoot(): JSX_2.Element;
+  tree: AppTree;
```

Comment:
Forgot about this bit and was thinking it could make sense to wrap things up a bit more potentially. Then I had the thought that maybe we should go in the opposite direction, where we only return `{ apis, tree }` here and grab the root from the tree instead. Or going even further, adding `apis` into the `tree` and just returning `{ tree }`

Either way, happy to merge as is but I might tinker a bit with this in a followup.